augur/translate.py

```diff
@@ -328,8 +347,7 @@ def sequences_vcf(reference_fasta, vcf):
     [0] The sequences as a dict of dicts. sequences → <NODE_NAME> → <POS> → <ALT_NUC> where <POS> is a 0-based int
     [1] The sequence of the provided `reference_fasta` (string)
     """
-    if not reference_fasta:
-        raise AugurError("A reference Fasta is required with VCF-format input")
+    assert(reference_fasta)
```

Comment:
All the other `assert`s in the codebase are strictly boolean without parentheses. Suggestion: make this more explicit, e.g.

```suggestion
    assert reference_fasta is not None
```

Reply:
Yeah I always forget  `assert` is a keyword. Fixed in force push.